calendar/v0.1.0/schema.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://common.schemas.example.com/calendar/v0.1.0/schema.json",
+  "title": "Calendar",
+  "titlePlural": "Calendars",
+  "description": "Google Calendar information",
+  "type": "object",
+  "appearance": {
+    "style": {
+      "color": "#42A5F5",
+      "icon": "./calendar-icon.svg"
+    }
+  },
+  "database": {
+    "name": "calendar_data",
+    "indexes": {
+      "id": [
+        "id"
+      ],
+      "summary": [
+        "summary"
+      ],
+      "timeZone": [
+        "timeZone"
+      ]
+    }
+  },
+  "layouts": {
+    "create": [
+      "summary",
+      "description",
+      "location",
+      "timeZone"
+    ],
+    "view": [
+      "id",
+      "summary",
+      "description",
+      "location",
+      "timeZone"
+    ]
+  },
+  "allOf": [
```

Comment:
See other schemas, they need to include the base verida schema.

See example here: https://github.com/verida/schemas-common/blob/92775e7dd3006c5fbf1edda5aa8cc78a4df2ddde/social/following/v0.1.0/schema.json#L38C71-L38C73